code/go/0chain.net/sharder/sharder/sharder.go

```diff
@@ -224,8 +224,10 @@ func main() {
 	go sc.StartLFMBWorker(ctx)
 
 	setupBlockStorageProvider(mConf, workdir)
-	sc.SetupGenesisBlock(viper.GetString("server_chain.genesis_block.id"),
-		magicBlock, initStates)
+	if sc.GetCurrentRound() == 0 {
+		sc.SetupGenesisBlock(viper.GetString("server_chain.genesis_block.id"),
+			magicBlock, initStates)
+	}
```

Comment:
This should have been fixed by latest staging code, please merge or rebase

Reply:
resolved